sway/desktop/output.c

```diff
@@ -1100,3 +1107,175 @@ void handle_output_power_manager_set_mode(struct wl_listener *listener,
 	oc = store_output_config(oc);
 	apply_output_config(oc, output);
 }
+
+static double lerp (double a, double b, double t) {
+	return a * (1.0 - t) + b * t;
+}
+
+static double ease_out_cubic (double t) {
+	double p = t - 1;
+	return pow(p, 3) + 1;
+}
```

Comment:
let's move these to a new file - I think they'll be important for when we add the ws movement animation